altos-rust/altos-core/src/arch/test.rs

```diff
@@ -26,13 +26,16 @@ pub fn yield_cpu() {
     sched::switch_context();
 }
 
-pub fn initialize_stack(stack_ptr: Volatile<usize>, _code: fn(&mut Args), _args: &Box<Args>) -> usize {
+pub fn initialize_stack(stack_ptr: Volatile<usize>, _code: fn(&mut Args), _args: &Box<Args>)
+    -> usize {
+
```

Comment:
Not really sure about having the return type on its own line like this. Maybe move one of the args down a line and have the return type off that?

```
pub fn initialize_stack(stack_ptr: Volatile<usize>, _code: fn(&mut Args),
                        _args: &Box<Args>) -> usize {
```